index.js

```diff
@@ -281,7 +281,17 @@ function redisStore(args) {
       if (err) {
         return cb && cb(err);
       }
-      conn.del(key, handleResponse(conn, cb));
+
+      if (Object.prototype.toString.call(key) === '[object Array]') {
+        var multi = conn.multi();
+        for (var i = 0, l = key.length; i < l; ++i) {
```

Comment:
key.length is't a function so it won't be called every time so i don't see how this is faster or am i missing something?